roles/arteria-sequencing-report-ws/files/nextflow_configs/demultiplex.config

```diff
@@ -0,0 +1,51 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+process {
+
+    withName: BCL2FASTQ {
+        //meta.lane ? "--tiles s_${meta.lane}" : "",
+        ext.args = {[
+            params.ext_args ? "${params.ext_args}" : "",
+        ].join(" ").trim()}
+        publishDir = [
+            [
+                path: { "${params.outdir}/Unaligned/" },
```

Comment:
Does this mean that all Fastqs will be stored directly in the Unaligned folder? I.e. not be sorted by project?

Reply:
The folder structure will be preserved. From one of my tests, this is how it looks like:
```
[adrienc@miarka3 Unaligned]$ tree
.
├── 200624_A00834_0183_BHMTFYDRXX
├── AB-1234
│   └── Sample_14574-Qiagen-IndexSet1-SP-Lane1
│       └── 14574-Qiagen-IndexSet1-SP-Lane1_S1_L001_R1_001.fastq.gz
├── Stats
│   ├── AdapterTrimming.txt
│   ├── ConversionStats.xml
│   ├── DemultiplexingStats.xml
│   ├── DemuxSummaryF1L1.txt
│   ├── FastqSummaryF1L1.txt
│   └── Stats.json
└── Undetermined_S0_L001_R1_001.fastq.gz
```

However, I just noticed the `200624_A00834_0183_BHMTFYDRXX` empty folder now, not sure where it comes from, will fix now.

Reply:
Not sure it's possible to do this easily though. From what I understand, this comes from the fact that the bcl2fastq work directory has a link to the runfolder, but since we place `Unaligned` folder directly in the runfolder, this creates a circular dependency. Nextflow is able to detect it and ignores the files from that folder, but somehow it still creates this empty folder.

I don't think it's a big deal though since this folder is empty. When I have time I can investigate further and either fix it or file an issue in the demultiplex repo.

Reply:
Ah I see :+1: Yeah, I agree, as long as it is empty is does not hurt (only the eyes a bit :hocho: :eyes:). Perhaps you can document this issue in a ticket?

Reply:
Done: https://snpseq.atlassian.net/browse/DATAOPS-676